src/content/en/updates/2017/06/headless-karma-mocha-chai.md

```diff
@@ -0,0 +1,137 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: "Testing on Headless Chrome with Karma/Mocha/Chai"
```

Comment:
When I see slashes I think that we're talking about optional things. So I thought the purpose of this article was "Testing on Headless with Karma, Mocha, or Chai"

So I suggest changing this to

"... with Karma, Mocha, and Chai"